base/compiler/tfuncs.jl

```diff
@@ -1713,7 +1713,7 @@ const _tvarnames = Symbol[:_A, :_B, :_C, :_D, :_E, :_F, :_G, :_H, :_I, :_J, :_K,
             end
```

Comment:
This if branch has a weird `isa(ai, Type)` instead of `ai = widenconst(ai)` call

Reply:
A brief look at the history reveals that it was added by some @martinholters in #27150 who apparently had no clue what he was doing, but merged it after @vtjnash had given his approval. :smile: